bokehjs/test/devtools/devtools.ts

```diff
@@ -752,6 +758,11 @@ async function run_tests(ctx: TestRunContext): Promise<boolean> {
       }
 
       if (failures != 0) {
+        if (num_selected_tests == num_all_tests) {
+          fail(`\n${chalk.cyan(failures)} of ${chalk.cyan(num_selected_tests)} tests ${chalk.red("failed")}`)
+        } else {
+          fail(`\n${chalk.cyan(failures)} of ${chalk.cyan(num_selected_tests)} selected tests ${chalk.red("failed")} (out of ${chalk.cyan(num_all_tests)} total tests)`)
+        }
```

Comment:
Maybe just provide all the numbers so any can be compared at a glanch
```
6 passing, 7 failed of 13 selected tests (15 skipped out of 28 total tests)
```

Reply:
👍 even better

Reply:
The output is now:

![image](https://github.com/user-attachments/assets/2da0c906-6fea-412f-93cb-5f47e431ec48)